client/deployment.go

```diff
@@ -97,16 +98,21 @@ func (d *Deployment) Status() (*http.Response, error) {
 // Reset shuts down deployment and deletes the contents of the deployment's
 // project directory
 func (d *Deployment) Reset() (*http.Response, error) {
-	return d.request("POST", "/reset", nil)
+	return d.post("/reset", nil)
 }
 
 // Logs get logs of given container
 func (d *Deployment) Logs(stream bool, container string) (*http.Response, error) {
-	reqContent := &common.DaemonRequest{
-		Stream:    stream,
-		Container: container,
+	//reqContent := &common.DaemonRequest{
+	//	Stream:    stream,
+	//	Container: container,
+	//}
+	reqContent := map[string]string{
+		"Stream": strconv.FormatBool(stream),
```

Comment:
isn't it standard for request parameters to be lowercased?

Reply:
done

Reply:
I know commenting "done" is pretty popular, but tbh just pushing a commit that fixes this is usually enough, GitHub will mark this code as changed 😆 idm either way though

Reply:
oh sorry, I usually do it to keep track and make sure I've addressed all comments! Are you getting emails every time I do that?

Reply:
I don't have email notifications enabled no, so it's all g either way 👍